truffle.js

```diff
@@ -41,17 +41,23 @@ module.exports = {
       gasPrice: 0,
       network_id: "1",
     },
+    mainnet: {
+      provider: () => {
+        return new HDWalletProvider("private-key", "https://mainnet.infura.io/v3/infura-key");
+      },
+      network_id: "1",
+    },
     goerli: {
       provider: () => {
-        return new HDWalletProvider("replace-with-private-key-when-using", "https://goerli.infura.io/v3/e21146aa267845a2b7b4da025178196d");
+        return new HDWalletProvider("private-key", "https://goerli.infura.io/v3/infura-key");
       },
       network_id: "5",
     },
-    mainnet: {
```

Comment:
I still want mainnet in this file.

Reply:
I moved it up the file, so the networks are ordered by id.